photoshop/api/__init__.py

```diff
@@ -31,6 +32,7 @@
 __all__ = [  # noqa: F405
     "ActionDescriptor",
     "ActionReference",
+    "ActionList",
```

Comment:
We also need to add `ActionList` in [`session.py`](https://github.com/loonghao/photoshop-python-api/blob/main/photoshop/session.py#L124)